src/patterns/data/default/entity_attribute.tsv

```diff
@@ -7242,3 +7242,12 @@ OBA:1001089	multicellular organism sex	UBERON:0000468	multicellular organism	PAT
 OBA:1001090	feather color	UBERON:0000022	feather	PATO:0000014	color
 OBA:1001091	egg shell trait	UBERON:0005079	eggshell	PATO:0000001	quality
 OBA:1001092	egg shell color	UBERON:0005079	eggshell	PATO:0000014	color
+OBA:2040143	metafemur length	HAO:0001140	metafemur	PATO:0000122	length
+OBA:2040144	beak length	UBERON:0005094	beak	PATO:0000122	length
+OBA:2040145	cell length	GO:0005623	cell	PATO:0000122	length
+OBA:2040146	cell width	GO:0005623	cell	PATO:0000921	width
+OBA:2040147	fore wing length	HAO:0000351	fore wing	PATO:0000122	length
+OBA:2040148	hind wing length	HAO:0000400	hind wing	PATO:0000122	length
+OBA:2040149	seed volume	PO:0009010	seed	PATO:0000918	volume
```

Comment:
remove this. this has to go into PO. Just drop, no further action required.

Reply:
Done.